src/SFA.DAS.EmployerIncentives.Api/Authorization/AuthorizationExtensions.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace SFA.DAS.EmployerIncentives.Api.Authorization
+{
+    public static class ApiAuthorizationExtensions
```

Comment:
As per previous, looks like this could be shared?

Reply:
agree